app/grants/views.py

```diff
@@ -493,6 +493,12 @@ def bulk_grants_for_cart(request):
     following = request.GET.get('following', '') != ''
     only_contributions = request.GET.get('only_contributions', '') == 'true'
 
+    try:
+        round_num = request.GET.get('clr_round')
+        clr_round = GrantCLR.objects.get(pk=round_num)
```

Comment:
there's an actual `round_num` field in `GrantCLR` model. so it looks confusing using `round_num` as a variable name here since it's actually the `clr_round_pk` that's sent as `clr_round` param.

maybe use `clr_round_pk` ?

consider using `

Reply:
addressed